README.md

```diff
@@ -187,6 +187,18 @@ Note that only `method`, `headers`, `redirect` and `body` are allowed in `window
 }
 ```
 
+#### Default Headers
+
+If no values are set, the following request headers will be sent automatically:
+
+Header            | Value                                                   
+----------------- | --------------------------------------------------------
+`Accept-Encoding` | `gzip,deflate`                                          
```

Comment:
This is done only when `compress` is true: https://github.com/bitinn/node-fetch/blob/560d5d6a023f069c37497f633fe65955182fa3d5/src/request.js#L145-L147